src/main/webapp/guide.jsp

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+    <a href="/activity">Activity</a>
+  </nav>
+
+  <div id="container">
+    <div
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+
+      <h1>How to style your text on the CodeU Chat App</h1>
+      <p>
+        Our chat app currently uses the BBCode markup language to allow users to syle their text. We currently do not allow HTML tags for text styling in order for our app to be as user friendly as possible. You can see the guide below that lists all the possible BBCode tags that you can use when starting a conversation on our chat app.
+      </p>
+      <br>
+            <table cellpadding = "15" cellspacing = "1" border="4">
+              <tr>
+                  <td><b>BBCODE</b></td>
+                  <td><b>Output</b></td>
+              </tr>
+              <tr>
+                  <td>[b]bolded text[/b]</td>
+                  <td><b>Bolded Text</b></td>
+              </tr>
+              <tr>
+                  <td>[i]italicized text[/i]</td>
+                  <td><i>italicized text</i></td>
+              </tr>
+              <tr>
+                  <td>[u]underlined text[/u]</td>
+                  <td><u>underlined text</u></td>
+              </tr>
+              <tr>
+                  <td>[s]strikethrough text[/s]</td>
+                  <td><s>strikethrough text</s></td>
+              </tr>
+              <tr>
+                  <td>[url]https://www.google.com/[/url]</td>
+                  <td><a href="https://www.google.com/">https://www.google.com/</a></td>
+              </tr>
+              <tr>
+                  <td>[url=https://www.google.com/]GOOGLE[/url]</td>
+                  <td><a href="https://www.google.com/">GOOGLE</a></td>
+              </tr>
+              <tr>
+                  <td>[img]https://tinyurl.com/y9cpmcor[/img]</td>
```

Comment:
In general you should be careful about depending on URLs like this that you may not necessarily control or have rights to. This is fine for now, but note that tinyurl.com can essentially track every time this page is loaded now and/or change what shows up on this page. You have to be super careful about img tags because it provides a way for a user to trigger a mandatory request for the image URL whenever another user tries to look at the message.